lvsfunc/deblock.py

```diff
@@ -165,44 +169,63 @@ def dpir(
+
+                                    This means you can pass higher strenghts to areas around edges with heavy ringing,
+                                    and lower in textured/detailed parts, for example.
+    :param mode:                    DPIR mode. Valid modes are 'deblock' and 'denoise'.
+    :param matrix:                  Enum for the matrix of the Clip to process.
+                                    See :py:attr:`lvsfunc.types.Matrix` for more info.
+                                    If not specified, gets matrix from the "_Matrix" prop of the clip
+                                    unless it's an RGB clip, in which case it stays as `None`.
+    :param cuda:                    Used to select backend.
+                                    Use CUDA if True, CUDA TensorRT if 'trt', else CPU OpenVINO if False.
+                                    If ``None``, it will detect your system's capabilities
+                                    and select the fastest backend.
+    :param i444:                    Forces the returned clip to be YUV444PS instead of the input clip's format.
+    :param tiles:                   Can either be an int, specifying the number of tiles the image will be split into
+                                    for processing, or a tuple for manually specifying the width/height
+                                    of the singular tiles.
+    :param overlap:                 Number of pixels in each direction for padding the tiles.
+                                    Useful for, when using tiled processing, you're having clear boundaries
+                                    between tiles.
+                                    To disable, set `None` or `0`. Default: 8px.
+    :param zones:                   A mapping to zone the DPIR strength so you don't have to call it multiple times.
+                                    The key should be a `float` / ``VideoNode`` (a normalised mask, for example)
+                                    or `None` to passthrough the Clip to process.
+                                    The values should be a range that will be passed to ``replace_ranges``
+    :param fp16:                    Represent the clip with 16f tensors instead of 32f for a speedup, but it's useless—
+                                    and may even harm performance—when enabled with a device that doesn't support it.
+    :param num_streams:             Number of concurrent CUDA streams to use. Increase if GPU isn't getting saturated.
+    :param device_id:               Specifies the GPU device id to use.
+    :param kernel:                  py:class:`vskernels.Kernel` object used for conversions between YUV <-> RGB.
+                                    This can also be the string name of the kernel
+                                    (Default: py:class:`vskernels.Bicubic(0, 0.5)`).
+
+    :return:                        Deblocked or denoised clip in either the given clip's format or YUV444PS.
+
```

Comment:
https://discord.com/channels/@me/856164891098742804/986762365141942442 👀

Reply:
Told you I'd forget